modules/oc-mirror-command-reference.adoc

```diff
@@ -107,12 +104,13 @@ The following tables describe the `oc mirror` subcommands and flags:
 |`--source-use-http`
 |Use plain HTTP for the source registry.
 
-|`--use-oci-feature`
-|Enable mirroring for local OCI catalogs on disk to the target mirror registry.
-
-The `--use-oci-feature` flag is deprecated. Use the `--include-local-oci-catalogs` flag instead.
-
 |`-v`, `--verbose` `<int>`
 |Specify the number for the log level verbosity. Valid values are `0` - `9`. The default is `0`.
 
 |===
+
+[NOTE]
+====
+The `--use-oci-feature` flag is deprecated. Use local OCI catalogs.
```

Comment:
I see `--use-oci-feature` and `--include-oci-catalogs` is deprecated if i am not wrong